python/graphstorm/eval/evaluator.py

```diff
@@ -381,67 +405,64 @@ def get_val_score_rank(self, val_score):
 
     @property
     def metric_list(self):
-        """ evaluation metrics
+        """ Return the evaluation metric list, which is given in class initialization.
         """
         return self._metric_list
 
     @property
     def best_val_score(self):
-        """ Best validation score
+        """ Return the best validation score of the major metric stored in this evaluator.
```

Comment:
Based on the implementation, it is a dict, which means the best validation score for each metric.

Reply:
Yes, updated to `Return the best validation score of metrics used in this evaluator.`